helm/app-exporter/templates/service-monitor.yaml

```diff
@@ -1,4 +1,3 @@
-{{- if .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" }}
 apiVersion: monitoring.coreos.com/v1
```

Comment:
@QuentinBisson if think since we are going to drop the job from PMO, we can make the SM unconditional, is that right?

Reply:
definitely

Reply:
Thanks! 

On the other hand I think this may be safer to leave the condition for SM, and also for the `giantswarm.io/monitoring` label in SVC, in case it gets deployed once for the cluster without `monitoring.coreos.com/v1`. In such case the `app-exporter` should become target for the `CLUSTER-prometheus/workload-CLUSTER/0` job. And then once it gets re-deployed with `monitoring.coreos.com/v1` in place, the label should be replaced for SM.

Reply:
WDYT?